### PR TITLE
018

### PR DESCRIPTION
**Pull request by KevinStolk**
Show is much smaller, .env.example is back and create is more polished.

### CODE REVIEW

File: app/Http/Controllers/CompanyController.php [hunk 1]
```diff
@@ -3,20 +3,27 @@
...
 use App\Http\Requests\CreateCompanyRequest;
 
 class CompanyController extends Controller
 {
+    /**
+     * Force user to be logged in before accessing any user information
+     */
+
+    public function __construct() {
+        $this->middleware('auth');
```
Comment (Evergetic) [inline app/Http/Controllers/CompanyController.php]: Middleware moet aangepast worden

**Review by Linskens1996 — APPROVED** [review]

File: app/Http/Controllers/CompanyController.php [hunk 2]
```diff
@@ -3,20 +3,27 @@
...
+
     /**
      * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
     public function index()
     {
-        $companies = Company::paginate(10);
+        $companies = Company::paginate(9);
```
Comment (oga-sven) [inline app/Http/Controllers/CompanyController.php:26]: why 9?

File: app/Http/Controllers/CompanyController.php [hunk 3]
```diff
@@ -28,6 +35,8 @@ public function index()
      */
     public function create()
     {
+        $this->middleware('auth');
```
Comment (oga-sven) [inline app/Http/Controllers/CompanyController.php:38]: wrm niet in construct?

File: routes/web.php [hunk 4]
```diff
@@ -22,8 +22,6 @@
...
 // Company
-Route::get('/company/create', 'CompanyController@create')->name('company.create')->middleware('auth');
-Route::post('/company/create', 'CompanyController@store');
-Route::get('/company/{company}', 'CompanyController@show')->name('company.show')->middleware('auth');
-Route::get('/companies', 'CompanyController@index')->name('company.index')->middleware('auth');
-Route::get('/company/search/{company}', 'CompanyController@search')->name('company.search')->middleware('auth');
\ No newline at end of file
+Route::resource('company', 'CompanyController');
+
+Route::get('/company/search/{company}', 'CompanyController@search')->name('company.search')
```
Comment (oga-sven) [inline routes/web.php:27]: waar is middleware auth?

**Review by oga-sven — APPROVED** [review]

**Review by oga-sven — APPROVED** [review]
bruh sound effect 45

**Review by Evergetic — APPROVED** [review]

File: app/Http/Controllers/CompanyController.php (hunk 2 above)
Comment (oga-sven) [inline app/Http/Controllers/CompanyController.php:26]: wrm 9!11!!111!@!@!@

**Review by Linskens1996 — APPROVED** [review]

File: app/Http/Controllers/CompanyController.php (hunk 3 above)
Comment (oga-sven) [inline app/Http/Controllers/CompanyController.php:38]: doe in construct dan

File: routes/web.php (hunk 4 above)
Comment (oga-sven) [inline routes/web.php:27]: wrm hier geen middleware wasmetjou

**Pull request merged**